orchagent/vnetorch.cpp

```diff
@@ -1972,3 +1972,96 @@ bool VNetRouteOrch::delOperation(const Request& request)
 
     return true;
 }
+
+VNetCfgRouteOrch::VNetCfgRouteOrch(DBConnector *db, DBConnector *appDb, vector<string> &tableNames)
+                                  : Orch(db, tableNames),
+                                  m_appVnetRouteTable(appDb, APP_VNET_RT_TABLE_NAME),
+                                  m_appVnetRouteTunnelTable(appDb, APP_VNET_RT_TUNNEL_TABLE_NAME)
+{
+}
+
+void VNetCfgRouteOrch::doTask(Consumer &consumer)
+{
+    SWSS_LOG_ENTER();
+
+    const string & table_name = consumer.getTableName();
+    auto it = consumer.m_toSync.begin();
+
+    while (it != consumer.m_toSync.end())
+    {
+        bool task_result = false;
+        auto t = it->second;
+        const std::string & op = kfvOp(t);
```

Comment:
could you remove the `std::` in the cpp file?

Reply:
remove them all for my changes in this .cpp files